10_checkboxes/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Checkboxes</title>
+  </head>
+  <body>
+    <div class="inbox">
+      <div class="item">
+        <input type="checkbox" />
+        <p>This is an inbox layout.</p>
```

Comment:
What tutorial did you follow for this? This is an incorrect way of creating accessible input fields.

Every input but have an associated label. There are [multiple ways](https://www.w3.org/WAI/tutorials/forms/labels/)  to do it. In this case, you could do:

```html
<label class="item">
  <input type="checkbox" />
  <span>This is an inbox layout.</span>
</label>
```

You'll notice the style will break. That's because `p` tag is being used to style an element. This why I tend to avoid using tags to style elements and prefer to use class. e.g. `.checkbox-text`.

Another advantage of using `<label>` is that now you the checkbox will be ticked even when clicking in the text.
 

Note:  `<p>` cannot be child of `<label>`. You can see the _Permitted content_ of any [element page under MDN page - Technical Summary](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)

Reply:
https://courses.wesbos.com/account/access/5eb4bad49edbdf36381229dc/view/194129765
This is from WesBos.
I will make a memo! thank you :)

Reply:
I can't see the page because I don't have an account. Could you take a few screenshots and send it to me over Discord?